### PR TITLE
PR Template includes changed js-types

### PR DESCRIPTION
**Pull request by weichweich**

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+- Step 1
+- Step 2
+- etc.
+
+### custom types
+
+This PR introduces new custom JS-types. Use the following to test the code with the polkadot apps:
+
+<detailts>
```
Comment (wischli) [inline .github/PULL_REQUEST_TEMPLATE.md:17]:
```suggestion
<details>
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+</details>
+
+## Checklist:
+
+- [ ] I have verified that the code works
+  - [ ] No panics! (checked arithmetic ops, no indexing `array[3]` use `get(3)`, ...)
+- [ ] I have verified that the code is easy to understand
+  - [ ] If not, I have left a well-balanced amount of inline comments
+- [ ] I have [left the code in a better state](https://deviq.com/boy-scout-rule/)
```
Comment (Diiaablo95) [inline .github/PULL_REQUEST_TEMPLATE.md:27]: 404 error?

**Review by wischli — CHANGES_REQUESTED** [review]
I would suggest we also enforce to open a PR in our [KILT types repository](https://github.com/KILTprotocol/type-definitions) with the type changes.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+Please provide a brief step-by-step instruction.
+If necessary provide information about dependencies (specific configuration, branches, database dumps, etc.)
+
+- Step 1
+- Step 2
+- etc.
+
+### custom types
+
+This PR introduces new custom JS-types. Use the following to test the code with the polkadot apps:
```
Comment (wischli) [inline .github/PULL_REQUEST_TEMPLATE.md:15]:
```suggestion
This PR introduces new custom JS-types which are required for compatibility with [our SDK](https://github.com/KILTprotocol/sdk-js) and the [Polkadot Apps](https://polkadot.js.org/apps/#/extrinsics). Please use the following types to test the code with the Polkadot Apps:
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+  {}
+  ```
+</details>
+
+## Checklist:
+
+- [ ] I have verified that the code works
+  - [ ] No panics! (checked arithmetic ops, no indexing `array[3]` use `get(3)`, ...)
+- [ ] I have verified that the code is easy to understand
+  - [ ] If not, I have left a well-balanced amount of inline comments
```
Comment (wischli) [inline .github/PULL_REQUEST_TEMPLATE.md:30]:
```suggestion
  - [ ] If not, I have left a well-balanced amount of inline comments
- [ ] This PR does not introduce new custom types
  - [ ] If not, I have opened a companion PR with the type changes in the [KILT types-definitions repository](https://github.com/KILTprotocol/type-definitions/pulls)
```

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (weichweich) [reply .github/PULL_REQUEST_TEMPLATE.md]: solved!

**Review by wischli — APPROVED** [review]

**Pull request merged**